frontend/src/scenes/sessions/filters/SavedFiltersMenu.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { FilterOutlined, PlaySquareOutlined, UserOutlined, EditOutlined } from '@ant-design/icons'
+import { SavedFilter, sessionsFiltersLogic } from 'scenes/sessions/filters/sessionsFiltersLogic'
+import { Button, Menu, Skeleton } from 'antd'
+import { Link } from 'lib/components/Link'
+import { toParams } from 'lib/utils'
+import { router } from 'kea-router'
+import { Drawer } from 'lib/components/Drawer'
+import { SaveFilter } from 'scenes/sessions/filters/SaveFilter'
+
+const ICONS: Record<SavedFilter['id'], JSX.Element | undefined> = {
+    all: <UserOutlined />,
```

Comment:
Not convinced on this icon for the all filter (i.e. no filter), any suggestions @corywatilo or @lottiecoxon ?